rpmio/rpmio.c

```diff
@@ -1100,7 +1101,12 @@ static rpmzstd rpmzstdNew(int fdno, const char *fmode)
 	    flags &= ~O_ACCMODE;
 	    flags |= O_RDWR;
 	    continue;
-	    break;
+	case 'T':
```

Comment:
I think it would be nice to support setting it to -1, same as for lzma:

```
                if (threads == -1)
                    threads = rpmExpandNumeric("%{getncpus}");
```